### DISCUSSION TITLE
Exobiology Sightseer milestone

### DISCUSSION THREAD

**Discussion by justind3d** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#45] [Q&A] [answered]
I'm having an issue with this expedition, phase 3 has the sightseer milestone to visit several very specific world types. all are triggering except for airless, i've been on 2 planets now that specifically are called 'airless' planet and the milestone is not checking off. could this be an NMS error or something with the scripting for the expedition json? I'm leaning more towards NMS but figured i'd check if anyone else had this issue as well. Thanks!

**Comment by justind3d** [comment]
ok, 5th airless planet surveyed finally triggered it, still not sure if it was me, NMS or the json but it finally checked off.

**Comment by cwmonkey** [comment, marked as answer]
I actually just did this one and it triggered on the first airless planet I hit, but I believe it was called a "life incompatible" planet. Chalk it up to a weird NMS bug that may have existed during the original expedition and they just decided to never fix it, or they added a new airless planet sub-type that doesn't trigger the milestone.

Oh, worth noting there are lots of names for "airless" planets, the main name is "Dead": https://nomanssky.fandom.com/wiki/Biome_-_Dead

They can show up as one of the following:

Terraforming Catastrophe
Dead
Empty
Desolate
Lifeless
Forsaken
Life-Incompatible
Low Atmosphere
Airless
Abandoned

All of those _should_ work.
